src/test/java/calculator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,106 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringAddCalculatorTest {
+    @Test
+    @DisplayName("Calculator는 null을 받을 경우 0을 반환한다")
+    void calculator_on_null() {
+        Integer sum = StringAddCalculator.splitAndSum(null);
+        assertThat(sum).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(sum).isZero();
```
0과 비교할 때 사용하는 더 적절한 대안이 있으니 참고바랍니다.

Reply:
훨씬 읽기 쉬운 방향인 것 같습니다. 제안해주신 방법으로 수정해두겠습니다 !